fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/responses/ViewHelpers.java

```diff
@@ -247,6 +248,25 @@ public String getObjectsAsString(final Graph graph,
     }
 
     /**
+     *
+     * @param graph
+     * @param subject
+     * @return
+     */
+    public int getNumChildren(final Graph graph, final Node subject) {
+        LOGGER.trace("Getting number of children: s:{}, g:{}", subject, graph);
+        final Iterator<Node> iterator = listObjects(graph, subject, HAS_CHILD_COUNT.asNode());
```

Comment:
Then better something like
```
Iterator<String> count = listObjects(graph, subject, HAS_CHILD_COUNT.asNode()).filterKeep(Node::isLiteral).mapWith(Node::getLiteralLexicalForm);
return count.hasNext() ? Integer.parseInt(count.next()) : 0;
```

instead of several times the length of `if-then`, but even better to switch the pattern. Also, that property should be internally typed, and there should be no need to cast into `Integer`, but that's another story.